### ISSUE TITLE
help content

### ISSUE THREAD

**Issue by periperipea** [rackspace/rack#290]
Hi,

I notice that the --blah commandline options is missing from the output of "rack help" - I am referring to these:

https://developer.rackspace.com/docs/rack-cli/globaloptions/

It would be great if they could be included.

Thanks

**Comment by jrperritt** [comment]
Yes, the current behavior is expected and desired. Running any command with `--h` or `--help` will show you only the options for that command/sub-command. I think it would be confusing to users to have the global options show up for the `rack --help` command since those flags are not valid options to the `rack` command.

To see the global options, you currently have (at least) 3 options
1. The documentation that you referenced
2. Calling help on any 4-word command (e.g. `rack servers instance list --help`)
3. Running `rack init` creates a `man` page that on Unix systems can be viewed with `man ./rack.1`. The global flags are in the `Description` section.

**Comment by periperipea** [comment]
I appreciate your quick response, I just wish you hadn't closed it without any feedback (and after only two days, on a weekend).

In response:

#2 is not obvious - no combination of <command> <subcommand> <action> mentions "--help".  It mentions "help, h" wich shows you just the same output after <command> or <subcommand> and doesn't work if you put it after <action>

Also, putting either nothing or "help" after <command> or <subcommand> produces the exact same output, there's no reason to assume "--help" is needed after <action>

In other words, the UI behaviour is inconsistent.   Two possible ways to improve this:
- replace --help with help for <action>
- add a line to the help output that says something like:
   "--help to list [flags]"

#3 Again, this is not obvious.  The output of "rack init" does not mention this, although it mentiones other stuff.  Suggest letting it create the file under /usr/local/share/man/ or at least asking where it should go.

Thanks

**Comment by annegentle** [comment]
Hi, I think we were very eager release week. :) Thanks for following up. 

@jrperritt I'd like to take on the "-- help to list [flags]" adding a line to the help output. I'll also update the `rack init` info. I'll keep the work going in https://github.com/rackspace/rack/pull/282

**Comment by periperipea** [comment]
Thank you, Anne!
